applications/search/src/pages/search-page/search-page.jsx

```diff
@@ -24,67 +28,59 @@ const ReactGA = require('react-ga');
 
 const browser = detect();
 
-export class SearchPage extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      showFilterModal: false
-    };
-
-    this.handleClearFilters = this.handleClearFilters.bind(this);
-    this.handleSearchSubmit = this.handleSearchSubmit.bind(this);
-    this.handleDatasetFilterThemes = this.handleDatasetFilterThemes.bind(this);
-    this.handleDatasetFilterAccessRights = this.handleDatasetFilterAccessRights.bind(
-      this
-    );
-    this.handleDatasetFilterPublisher = this.handleDatasetFilterPublisher.bind(
-      this
-    );
-    this.handleDatasetFilterPublisherHierarchy = this.handleDatasetFilterPublisherHierarchy.bind(
-      this
-    );
-    this.handleDatasetFilterProvenance = this.handleDatasetFilterProvenance.bind(
-      this
-    );
-    this.handleDatasetFilterSpatial = this.handleDatasetFilterSpatial.bind(
-      this
-    );
-    this.handleFilterFormat = this.handleFilterFormat.bind(this);
-
-    this.handlePageChange = this.handlePageChange.bind(this);
+export const SearchPage = props => {
```

Comment:
kan det være mulig å ha stateless konversion i en ant commit? for mye ting skjer samtidlig

Reply:
opprettet egen commit for det